src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/SomaticGenotypingEngine.java

```diff
@@ -297,6 +262,41 @@ public CalledHaplotypes callMutations (
         return new CalledHaplotypes(outputCalls, calledHaplotypes);
     }
 
+    private void addStrandBiasAnnotationsAndFilter(Map<String, Integer> originalNormalReadQualities, VariantContext mergedVC, PerReadAlleleLikelihoodMap tumorPRALM, PerAlleleCollection<Double> altAlleleFractions, Allele alleleWithHighestTumorLOD, VariantContextBuilder callVcb) {
```

Comment:
I'm uncomfortable with the number of arguments some of these extracted methods have -- it's a code smell, for sure -- but it's a start.